devroom-settings/devroom_settings/models.py

```diff
@@ -63,7 +63,7 @@ def save(self, *args, **kwargs):
                     self.manager_team.save()
                 # mail should be stable from now on
                 # self.mail = f"{self.slug}-devroom-manager@fosdem.org"
-        if self.cfp_url and self.cfp_url != "":
+        if self.cfp_url and self.cfp_url != "" and str(self.track.description) == "":
```

Comment:
This is a bit of a hack to preserve the original default description but allow it to be modified afterwards